src/settings/tabcpp.cpp

```diff
@@ -0,0 +1,149 @@
+#include "tabcpp.h"
+
+#include <QtWidgets>
+#include <QAbstractItemView>
+
```

Comment:
Header QtWidgets includes all elements of Qt UI
It is huge
You must remove this include from your files, especially - from headers
**_It might cause identifiers conflict_**

Reply:
removed